dissect/target/helpers/record.py

```diff
@@ -144,14 +144,14 @@ def DynamicDescriptor(types):  # noqa
 )
 
 COMMON_INTERFACE_ELEMENTS = [
+    ("stringlist", "sources"),
```

Comment:
I'd just let the source be the single string to `HKLM\\SYSTEM\\CurrentControlSet\\Control\\Class\\{4d36e972-e325-11ce-bfc1-08002be10318}\\{key_name}` for simplicity. As that's the leading device key, and the rest is just additional metadata.